evento/src/views/MyEvents/index.js

```diff
@@ -1,14 +1,38 @@
 import React, { Component } from 'react';
-import './MyEvents.css'
+import Cookie from 'js-cookie';
+import EventCard from '../../components/EventCard';
+import './MyEvents.css';
 
 class MyEvents extends Component {
-	componentWillMount() {
+	constructor() {
+		super();
+		this.state = { events: [] };
+	}
+
+	componentDidMount() {
+		const userId = Cookie.get('userId');
+		const authToken = Cookie.get('auth_token');
+
+		fetch(`/users/${userId}/events`, {
```

Comment:
fetch is directed to `/users/undefined/events` and Authorization header undefined. So in essence the fetch fails and state.events will stay empty.